### ISSUE TITLE
False positive for CKV_AWS_143 ("Ensure S3 bucket has lock configuration enabled by default")

### ISSUE THREAD

**Issue by bogdanbarna** [bridgecrewio/checkov#1208]
**Describe the bug**

Checkov expects argument `object_lock_configuration` to be an object, i.e.

```hcl
object_lock_configuration = {
  object_lock_enabled = "Enabled"
}
```

Terraform works with the above configuration, but when also declaring rules for the object lock configuration, it expects a block instead, e.g.
```hcl
object_lock_configuration {
  object_lock_enabled = "Enabled"

  rule {
    default_retention {
      mode = "GOVERNANCE"
      days = 366
    }
  }
}
```

**Expected behavior**
Checkov should pass for a `object_lock_configuration` argument block.

**Desktop (please complete the following information):**
 - OS: macOS Big Sur 11.3.1
 - Checkov Version: 2.0.135
 - Terraform version: v0.14.8
